README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Specify the environment where these commands should run (e.g., macOS, Linux) to ensure clarity and prevent potential issues for users on different OS.